consensus/consensus_v2.go

```diff
@@ -209,6 +209,22 @@ func (consensus *Consensus) finalCommit() {
 	}
 
 	block.SetCurrentCommitSig(commitSigAndBitmap)
+
+	commitSigBitmap := CommitSigBitmaps{CommitSigBitmap: commitSigAndBitmap}
+	parentHeader := consensus.Blockchain.GetBlockByHash(block.ParentHash())
+	if consensus.Blockchain.Config().IsExtraCommit(parentHeader.Epoch()) && parentHeader.Number().Uint64() > 1 {
```

Comment:
Why are you using the condition `parentHeader.Number().Uint64() > 1` ?

Reply:
Added "// Extra commit sig logic only works starting from block 2 since genesis block doesn't have signatures."